WordPress/src/main/java/org/wordpress/android/ui/posts/PostUtils.java

```diff
@@ -364,4 +366,9 @@ public static void trackGutenbergDialogEvent(AnalyticsTracker.Stat stat, PostMod
         properties.put("is_page", post.isPage());
         AnalyticsUtils.trackWithSiteDetails(stat, site, properties);
     }
+
+    public static boolean shouldShowGutenbergEditor(boolean isNewPost, PostModel post) {
+        return AppPrefs.isGutenbergEditorEnabled()
+               && (isNewPost || contentContainsGutenbergBlocks(post.getContent()));
```

Comment:
Commenting here and we can open a ticket later:
It seems that we might need to also check for empty content and launch Gutenberg:
1. Create a new post with Gutenberg enabled
2. Gutenberg will launch
3. Put in a title but leave the content completely empty. Go back
4. A draft will be created
5. Open the draft. Notice that Aztec is used as the editor

I think this is counter-intuitive and Gutenberg should be launched again. WDYT @mzorz ?

Reply:
I agree that it should launch Gutenberg for empty posts.

Currently iOS is launching Aztec, same as Android. When this change gets implemented, please let us know to coordinate and update iOS too.

Reply:
agreed @hypest @etoledom 
Issues opened:
- WPiOS: https://github.com/wordpress-mobile/WordPress-iOS/issues/10647
- WPAndroid: https://github.com/wordpress-mobile/WordPress-Android/issues/8776